nss-day-cohort-46/sprinkles-of-joy-kylo-ren-s-rice-krispie-treats
# Lw initial debugging

# Description
Fixed our provider components. They werent originally returning anything.
Fixes # (issue)

## Type of change
Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)


# Testing Instructions

`git fetch --all`
`git checkout LW-initial-debugging`
 make sure there are no new errors.

# Checklist:
- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
